components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -188,7 +188,13 @@ func main() {
 	internalEvalAssistant := avs.NewInternalEvalAssistant(cfg.Avs)
 	externalEvalCreator := provisioning.NewExternalEvalCreator(avsDel, cfg.Avs.Disabled, externalEvalAssistant)
 
-	bundleBuilder := ias.NewBundleBuilder(httpClient, cfg.IAS)
+	var clientHTTPForIAS *http.Client
+	if cfg.IAS.TLSRenegotiationEnable {
+		clientHTTPForIAS = httputil.NewRenegotiationTLSClient(30, cfg.Director.SkipCertVerification)
+	} else {
+		clientHTTPForIAS = httpClient
+	}
```

Comment:
**suggestion**
```suggestion
	clientHTTPForIAS := httpClient
	if cfg.IAS.TLSRenegotiationEnable {
		clientHTTPForIAS = httputil.NewRenegotiationTLSClient(30, cfg.Director.SkipCertVerification)
	}
```

maybe it will be simpler to have a default and override with flag enabled? but I don't have a strong opinion about that if/else statements